label_studio/data_manager/aggregates.py

```diff
@@ -0,0 +1,81 @@
+"""
+Shim to address None values in arrays
+Fixed on https://github.com/django/django/commit/3dc9f3ac6960c83cd32058677eb0ddb5a5e5da43#diff-b6a6632418c964155865691be58c9f76717ef512ced704d461f41cc3612a1db3R240
+Remove this shim once this code is a required dependency
+"""
+from django import __version__
+
+from django.db.models.lookups import Exact
+from django.db.models import Func, Value
+from django.contrib.postgres import lookups
+from django.contrib.postgres.aggregates import ArrayAgg as OriginalArrayAgg
+from django.contrib.postgres.fields import ArrayField
+from django.contrib.postgres.fields.array import ArrayLenTransform as OriginalArrayLenTransform, ArrayInLookup as OriginalArrayInLookup
+
+
+class ArrayRHSMixin:
```

Comment:
I would solve it using another `if` statement here: https://github.com/heartexlabs/label-studio/blob/develop/label_studio/data_manager/managers.py#L195

Will it correctly work it with `[]` (empty list), when annotation.result = [] (instead of annotation.result = None) ?

Reply:
I don't want to make too many changes to this file. It's just a "fast forward" of aggregates.py to a newer django version without requiring the latest django version. I think it completely addresses the bug we're trying to solve as is.

Reply:
I still have two concerns here:
1. performance, how will it be affected? 
2. from user perspective `annotation.result = []` is empty annotation (you can try to get this by clicking submit without any region creation). However, this approach will process [] as not empty annotation.